.expeditor/verify.pipeline.yml

```diff
@@ -17,18 +17,32 @@ steps:
   expeditor:
     executor:
       docker:
-        image: ruby:2.5-buster
+        image: ruby:2.5
```

Comment:
ruby 2.5 has reached eol stage. Do we still need it here ?

Reply:
No, I was just checking couple of scenarios with 2.5. Removed the ruby 2.5 now